hazelcast-client/src/main/java/com/hazelcast/client/spi/ClientProxy.java

```diff
@@ -101,6 +105,17 @@ protected final HazelcastClientInstanceImpl getClient() {
         return (HazelcastClientInstanceImpl) getContext().getHazelcastInstance();
     }
 
+    protected final int getConnectedServerVersion() {
+        HazelcastClientInstanceImpl client = getClient();
+        ClientConnectionManager connectionManager = client.getConnectionManager();
+        Collection<ClientConnection> activeConnections = connectionManager.getActiveConnections();
```

Comment:
Previously we were checking the server version on the owner connection. Why was this changed to lookup server version on any active connection?

Reply:
Actually it is not required to be owner. On client side, even if you have a healthy connection it is not guaranteed that owner is decided even on a one node cluster. Decision of an owner connection is another parallel process. (Maybe this is a bug but this is the current situation) So using one of existing connection should be ok.